site/content/blog/2023-10-12-maplocal-add-on/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Map Local Add-on"
+summary: "Allows mapping of responses to content of chosen local file."
+images:
+#- https://www.zaproxy.org/blog/2023-10-12-maplocal-add-on/images/image.png
+type: post
+tags:
+- blog
+- maplocal
+date: "2023-10-12"
```

Comment:
We're publishing a blog post about 2.14.0 today, so it might be worth holding off for a couple of days 😉